components/prism-cmake.js

```diff
@@ -0,0 +1,16 @@
+	'boolean': /\b(?:ON|OFF|TRUE|FALSE)\b/,
+	'namespace': /\b(?:PROPERTIES|SHARED|PRIVATE|STATIC|PUBLIC|INTERFACE|TARGET_OBJECTS)\b/,
+	'operator': /\b(?:NOT|AND|OR|MATCHES|LESS|GREATER|EQUAL|STRLESS|STRGREATER|STREQUAL|VERSION_LESS|VERSION_EQUAL|VERSION_GREATER|DEFINED)\b/,
+	'number': /\b\d+(?:\.\d+)*\b/,
+	'function': /\b[a-z_]\w*(?=\s*\()\b/,
```

Comment:
This needs an `i` flag, otherwise, we don't match `foobar()`.

Reply:
How is it possible?
foobar is matched without this flag in the function_feature.test.

Reply:
Sorry, I mistyped. Fixed it.

Reply:
@mjrogozinski The `i` flag is necessary, i.e. `Foobar()`.

Reply:
`i` means case insensitive. Now I get it.
I need to learn more about those regex things.